meetings/2022-09-08.md

```diff
@@ -44,51 +46,72 @@
+* SC: Do you think the current text meets that?
+* AC: If I'm understanding this correctly, if you want to use --- funny requirement --- then you must do it this way. That's fine. Usually in other specs, if you want to enable, then you need to do x, y, z. Here we are saying If you want to do a, then do a, seems a little odd. One thing I can suggestion.. when would a server not want to enable this. I think the first one is a must. For the second one related to storage, that's where I think the current text is probably good. It doesn't make a hard requirement, but this is how you do it.
+* SC: Could we update the requirement for storageDescription?
+* RG: If I'm accessing a resource and get describedby, then that's extra info that I can follow that.
+* RG: Suppose there isn't a describedby, and only storageDescription, I would think that would be applicable for all the other resources on the same storage. only when there is a deviation, that'd when I would expect a describedby. but when there is a describedby, I would get that. there would be an extra fetch any way.
+* SC: so with my dev hat on: if I see describedby, I would get it and if I find notifications channel info, good enough for me. I stop. if I don't see the link header or don't find the info I want, then ill look into storage description link if there is one.. and follow to its description and so on.
+* SC: That may cover majority of implementations but I don't have data to back that up.
+* RG: if described by why would there be storage description
+* AC: say generic term like `rdfs:label` and put some info there but then you could also have more specific property like `skos:prefLabel` which might have the same info. but what this means is that you don't know what clients may be interacting.. some clients may not know SKOS but then they'd use the RDFS label. But other clients may like the more precise term. that puts extra burden on server but making the interop more widely available.
+* SC: Proposal:
+
+PROPOSAL: Keep the requirement as is. We can drop that when we get some implementation feedback from clients.
+
+* AC: +1
+* RG: +1
+
+* RG: AC, you are saying that storage description would have more details data. isn't that more of a generic thing to do.
+* AC: I'm saying the semantics of storage description (relation) is more precise than describedby.
+* RG: I would expect more specific semantics for a resource.
+* SC: so instead of describedby, more like `notify:notificationResource` ?
+* AC: I think the idea is that the describedby is going to be specific to a resource, and storage description is for the entire storage. there are two possibilities. the information follows the links. the other is that thye are different. in the case where they are different, and in the case with describedby, it would have more information but that would up to the implementation in the end.
```

Comment:
```suggestion
* SC: so instead of `describedby`, more like `notify:notificationResource`?
* AC: I think the idea is that the `describedby` is going to be specific to a resource, and storage description is for the entire storage. There are two possibilities. The information follows the links. The other is that they are different. In the case where they are different, and in the case with `describedby`, it would have more information, but that would be up to the implementation, in the end.
```